android/app/src/main/java/com/google/samples/cronet_sample/MainActivity.java

```diff
@@ -123,4 +113,50 @@ public void run() {
             this.cronetLatency.set(averageLatency);
         }
     }
+
+    private void setCronetEngine() {
+        // create the Cronet engine.
+        CronetEngine.Builder myBuilder = new CronetEngine.Builder(this);
```

Comment:
nit: join the two?

    cronetEngine = new CronetEngine.Builder(this).enableHttp...